lib/constellation.rb

```diff
@@ -0,0 +1,39 @@
+class ConstellationHttpException < StandardError
+end
+
+require 'net/http'
+module Constellation
+  def self.create_rsvp!(rsvp_params)
+    return unless constellation_enabled?
+    uri = URI.parse("#{app}/rsvps")
+    http = Net::HTTP.new(uri.host, uri.port)
+    unless Rails.env.development?
+      http.use_ssl = true
+      http.verify_mode = OpenSSL::SSL::VERIFY_NONE
```

Comment:
Ah ok. The `unless Rails.env.development?` conditional was throwing me off, I thought that if this was about running the apps locally, they would be running in the dev env.